.github/workflows/release-please-enforcer.yaml

```diff
@@ -0,0 +1,23 @@
+name: 'Release Please Enforcer'
```

Comment:
Can we change the name of the workflow to something like lint-pull-request.yaml / "Lint PR"?